app/assets/javascripts/views/common/tableView.js

```diff
@@ -0,0 +1,174 @@
+(function(App) {
+
+  'use strict';
+
+
+  App.View.TableView = Backbone.View.extend({
+
+    defaults: {
+      resultsPerPage: 3
+    },
+
+    el: '#table',
+
+    events: {
+      'click .pagination-link': '_updatePagination',
```

Comment:
This comment applies for the next two lines too.

We usually use `.js-*` classes to bind listeners. Also, let's replace the id tags because it could suggest we will style the elements with ids (which is not permitted with the sass-lint configuration). Classes starting with `.js-` shouldn't have styles.